evennia/commands/cmdsethandler.py

```diff
@@ -351,6 +351,14 @@ def update(self, init_mode=False):
                     elif path:
                         cmdset = self._import_cmdset(path)
                         if cmdset:
+                            if cmdset.key == '_CMDSET_ERROR':
+                                # If a cmdset fails to load, check if we have a fallback path to use
+                                fallback_path = settings.CMDSET_FALLBACKS.get(path, None)
```

Comment:
There's some error printing out in the `self._import_cmdset(path)` call that sets it to be `_CMDSET_ERROR` - do you think that's sufficient, or we should have more? When I have in-game errors on, I actually get the stack trace twice for some reason, not sure why yet.